python/pfs/drp/stella/fitBroadbandSED.py

```diff
@@ -17,8 +17,26 @@ class FitBroadbandSEDConfig(Config):
     filterMappings = DictField(
         keytype=str, itemtype=str,
         default={
-            "g": "HSCg", "r": "HSCr", "r2": "HSCr2", "i": "HSCi", "i2": "HSCi2",
-            "z": "HSCz", "y": "HSCy"
+            "g_hsc": "HSCg",
+            "r_old_hsc": "HSCr",
+            "r2_hsc": "HSCr2",
+            "i_old_hsc": "HSCi",
+            "i2_hsc": "HSCi2",
+            "z_hsc": "HSCz",
+            "y_hsc": "HSCy",
+            "g_ps1": "PS1g",
+            "r_ps1": "PS1r",
+            "i_ps1": "PS1i",
+            "z_ps1": "PS1z",
+            "y_ps1": "PS1y",
+            "bp_gaia": "GaiaBp",
+            "rp_gaia": "GaiaRp",
+            "g_gaia": "GaiaG",
+            "u_sdss": "SDSSu",
+            "g_sdss": "SDSSg",
+            "r_sdss": "SDSSr",
+            "i_sdss": "SDSSi",
+            "z_sdss": "SDSSz",
```

Comment:
Why do we have a two-step mapping from pfsConfig to internal name to filename, rather than a one-step mapping?

Reply:
The package (`fluxmodeldata-ambre...`) used by `fitBroadbandSED.py` is not version-controlled. I don't want to variegate it unnecessarily. I have an undecided plan to change `fluxmodeldata-ambre...` in near future (to add theoretically whitened flux), and if it happens, I will change the filter names used in `fluxmodeldata-ambre...` so that I can delete this mapping.